src/column/NumberColumn.ts

```diff
@@ -31,7 +31,7 @@ export default class NumberColumn extends AVectorColumn<number, INumericalVector
   constructor(data: INumericalVector, orientation: EOrientation, $parent: d3.Selection<any>, matrixCol?: MatrixColumn) {
     super(data, orientation);
     this.$node = this.build($parent);
-    this.attachListener();
+    //  this.attachListener();
```

Comment:
This event was responsible creating new matrix view from the converted vector only.So to make consistent as described in issue. I turned off the listener.But we can restore whole matrix from the conversion icon available on the matrix header.
![2017-06-16_09-36-05](https://user-images.githubusercontent.com/12292333/27216770-5ae897ea-5277-11e7-8d38-01c1b3fb8e3d.gif)